assets/app/scripts/services/data.js

```diff
@@ -708,7 +706,7 @@ angular.module('openshiftConsole')
 
   DataService.prototype._urlForType = function(type, id, context, isWebsocket, params) {
```

Comment:
log a warning if we get a non-lowercased type, but go ahead and lowercase the type